lib/wallet/integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -353,18 +354,142 @@ spec = describe "BYRON_TRANSACTIONS" $ do
                     (.>= Quantity (faucetAmt - faucetUtxoAmt))
             ]
 
-        eventually "wa and wb balances are as expected" $ do
-            rb <- request @ApiWallet ctx
+        let txId = getFromResponse (#id) r
+        eventually "wa and wb balances are as expected as well as fees on txs" $ do
+            -- check that tx and balance on source Byron wallet is as expected
+            rTxSrc <- request @(ApiTransaction n) ctx
+                (Link.getTransaction @'Byron wByron (ApiTxId txId)) Default Empty
+            verify rTxSrc
+                [ expectSuccess
+                , expectField (#amount . #getQuantity) (`shouldBe` amt + feeEstMax)
+                , expectField (#fee . #getQuantity) (`shouldBe` feeEstMax)
+                , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
+                ]
+            rSrc <- request @ApiByronWallet ctx
+                (Link.getWallet @'Byron wByron) Default Empty
+            verify rSrc
+                [ expectSuccess
+                , expectField
+                        (#balance . #available)
+                        (`shouldBe` Quantity (faucetAmt - feeEstMax - amt))
+                ]
+
+            -- check that tx and balance on destination Shelley wallet is as expected
+            rTxDest <- request @(ApiTransaction n) ctx
+                (Link.getTransaction @'Shelley wShelley (ApiTxId txId)) Default Empty
+            verify rTxDest
+                [ expectSuccess
+                , expectField (#amount . #getQuantity) (`shouldBe` amt)
+                , expectField (#fee . #getQuantity) (`shouldBe` feeEstMax)
```

Comment:
```suggestion
                , expectField (#fee . #getQuantity) $
                    between (feeEstMin, feeEstMax)
```

Now the destination wallet gets precisely `amt` of funds, but the fee is still only an estimate.